tools/proofers/PPage.inc

```diff
@@ -129,8 +129,7 @@ class PPage
             . "id={$this->lpage->projectid}"
             . $amp
             . "expected_state={$this->proj_state}"
-            . $amp
-            . "detail_level=1#project-comments";
+            . "#project-comments";
```

Comment:
This is the only change I'm wary about. Having the page at level 1 gives much easier access to the project comments which is the goal here.

Reply:
Fwiw, one of the complaints was from someone who didn't appreciate being sent to level 1 by clicking on the Project Comments link in the PI. I suspect that some people, anyway, have gotten used to thinking of it as an easy link back to the project page in general.

Reply:
I see I wasn't quite clear enough. I have, multiple times while testing, clicked on the "Project Comments" link from the project page. I *always* switch away from detail level 1, because what I want is below the Project Comments. We actually put the internal `#project-comments` link in to focus directly on the PCs, because even at detail level 1, you had to scroll down to get to them. You've retained the internal link directly to the PCs, so I think it'll be okay.